src/Components/AlbumCard/AlbumCard.css

```diff
@@ -11,7 +11,7 @@
   background: radial-gradient(ellipse at 13% 0, rgba( 205, 214, 221, 0.25 ) 25%, transparent 45%, rgba( 205, 214, 221, 0.25 ));
   backdrop-filter: blur( 1px );
   -webkit-backdrop-filter: blur( 1px );
-  transition: transform 1s;
+  transition: transform 2s;
```

Comment:
UX: transform time may be a little too slow. ~.6s feels more responsive.

Reply:
change made